test/functionTest.php

```diff
@@ -0,0 +1,18 @@
+<?php
+require('functions.php');
+use PHPunit\Framework\TestCase;
+
+class functionTest extends TestCase {
+
+    public function testSuccessloopCartoons() {
```

Comment:
what happens if your array has the wrong keys in it? no test for that

Reply:
Is the function I added correct? It was an educated guess that displayCartoons(6) would throw an error because (6) doesn't exist.